actions/admin_actions.jsx

```diff
@@ -333,3 +333,14 @@ export async function testSmtp(success, error) {
         error({id: err.server_error_id, ...err});
     }
 }
+
+
+export async function testSiteURL(success, error) {
```

Comment:
I think a better approach would be to remove line 339 and pass siteURL as a parameter to the action.